metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1029,7 +1030,8 @@ TimelineHashSet<TopicIdPartition> imbalancedPartitions() {
     }
 
     boolean isElrEnabled() {
-        return eligibleLeaderReplicasEnabled && featureControl.metadataVersion().isElrSupported();
+        return featureControl.metadataVersion().isElrSupported() && featureControl.finalizedFeatures(offsetControl.lastStableOffset()).
```

Comment:
I don't think this is correct. Why would we want isElrEnabled to be reading from a point in the past? Surely we want to know whether elr is CURRENTLY enabled, right?

Reply:
Updated to use the latest finalized version.